src/runtime/serverlet.h

```diff
@@ -90,6 +90,8 @@ class rpc_replier
         return _response != nullptr ? _response->to_address : rpc_address::s_invalid_address;
     }
 
+    int error_code() const { return _response != nullptr ? _response->error() : 0; }
```

Comment:
For `rpc_replier`, the reason why we just need the error code in `message_ex`, rather than field `error` in thrift object `TResponse`, is that there is no read api (`on_get`, `on_multi_get`, ...) using `rpc_replier` to reply ?

Reply:
Only simple_kv use this register function, simple_kv is just a simple demo of rdsn framework, so I'll update the code to limit the usage of function
```
template <typename TReq, typename TResp>
static bool register_async_rpc_handler(...)
```